decidim-core/app/commands/decidim/authorize_user.rb

```diff
@@ -34,5 +35,19 @@ def create_authorization
         metadata: handler.metadata
       )
     end
+
+    def unique?
+      return true if handler.unique_id.nil?
+
+      duplicates = Authorization.where(
+        user: User.where(organization: handler.user.organization.id),
+        name: handler.handler_name,
+        unique_id: handler.unique_id
+      )
```

Comment:
Hound won't let me do it but I changed it by `unless duplicates.any?` which is more readable.